### ISSUE TITLE
[BUG] 2.3.10 ResponsiveGridLayout: `cols` entry for breakpoint null is missing!

### ISSUE THREAD

**Issue by oneart-dev** [jbaysolutions/vue-grid-layout#487]
**Software version (please complete the following information):**
 - Chrome 86.0.4240.75
 - Vue Version 2.5.17
 - vue-grid-layout Version: 2.3.10

I updated package and got this error instantly.

**To Reproduce**
```js
<grid-layout
                    v-if="layout && !isMobile"
                    :layout="layout"
                    :col-num="colNum"
                    :row-height="rowHeight"
                    :is-draggable="true"
                    :is-resizable="true"
                    :is-mirrored="false"
                    :vertical-compact="true"
                    :margin="[10, 10]"
                    :use-css-transforms="false"
                    :responsive="true"
                    :autoSize="true"
                    @layout-updated="layoutUpdatedEvent"
                    @breakpoint-changed="breakpointChangedEvent"
                >
                    <grid-item
                        v-for="(item,index) in layout"
                        :static="item.static || globalStatic"
                        :x="item.x"
                        :y="item.y"
                        :w="item.w"
                        :h="item.h"
                        :i="item.i"
                        :minW="item.minW"
                        :minH="item.minH"
                        :key="item.i"
                        @resized="item.resized = Date.now()"
                    >
                        <widget
                            v-if="item"
                            :h="item.h"
                            :w="item.w"
                            :resized="item.resized"
                            :global-static="globalStatic"
                            :id="item.i" />
                    </grid-item>
                </grid-layout>
```

**Additional context**

```js
Error in mounted hook: "Error: ResponsiveGridLayout: `cols` entry for breakpoint null is missing!"

found in

---> <GridItem>
       <GridLayout>
         <Dashboard> at resources/js/components/views/account/Dashboard.vue
           <VMain>
             <VApp>
               <Layout> at src/layouts/full-layout/Layout.vue
                 <VApp>
                   <App> at resources/js/components/App.vue
                     <Root>
./resources/js/app.js.vue__WEBPACK_IMPORTED_MODULE_3___default.a.config.warnHandler @ app.js:111
warn @ vue.common.dev.js:628
logError @ vue.common.dev.js:1889
globalHandleError @ vue.common.dev.js:1884
handleError @ vue.common.dev.js:1844
invokeWithErrorHandling @ vue.common.dev.js:1867
callHook @ vue.common.dev.js:4216
insert @ vue.common.dev.js:3142
invokeInsertHook @ vue.common.dev.js:6337
patch @ vue.common.dev.js:6554
Vue._update @ vue.common.dev.js:3945
updateComponent @ vue.common.dev.js:4063
get @ vue.common.dev.js:4474
run @ vue.common.dev.js:4549
flushSchedulerQueue @ vue.common.dev.js:4307
(anonymous) @ vue.common.dev.js:1985
flushCallbacks @ vue.common.dev.js:1911
Promise.then (async)
timerFunc @ vue.common.dev.js:1938
nextTick @ vue.common.dev.js:1995
queueWatcher @ vue.common.dev.js:4399
update @ vue.common.dev.js:4539
notify @ vue.common.dev.js:741
reactiveSetter @ vue.common.dev.js:1066
(anonymous) @ vue-router.esm.js:2919
(anonymous) @ vue-router.esm.js:2918
updateRoute @ vue-router.esm.js:2339
(anonymous) @ vue-router.esm.js:2193
(anonymous) @ vue-router.esm.js:2327
step @ vue-router.esm.js:1931
step @ vue-router.esm.js:1938
step @ vue-router.esm.js:1938
runQueue @ vue-router.esm.js:1942
(anonymous) @ vue-router.esm.js:2322
step @ vue-router.esm.js:1931
(anonymous) @ vue-router.esm.js:1935
(anonymous) @ vue-router.esm.js:2309
(anonymous) @ vue-router.esm.js:2057
(anonymous) @ vue-router.esm.js:2133
Promise.then (async)
(anonymous) @ vue-router.esm.js:2080
(anonymous) @ vue-router.esm.js:2101
(anonymous) @ vue-router.esm.js:2101
flatMapComponents @ vue-router.esm.js:2100
(anonymous) @ vue-router.esm.js:2036
iterator @ vue-router.esm.js:2287
step @ vue-router.esm.js:1934
step @ vue-router.esm.js:1938
step @ vue-router.esm.js:1938
(anonymous) @ vue-router.esm.js:1935
(anonymous) @ vue-router.esm.js:2309
(anonymous) @ vue-router.2.x.js:36
_processTransitionEach @ vue-auth.esm.js:580
(anonymous) @ vue-router.2.x.js:34
Promise.then (async)
_processAuthenticated @ vue-auth.esm.js:537
(anonymous) @ vue-auth.esm.js:525
Promise.then (async)
_processRouterBeforeEach @ vue-auth.esm.js:524
(anonymous) @ vue-router.2.x.js:31
iterator @ vue-router.esm.js:2287
step @ vue-router.esm.js:1934
(anonymous) @ vue-router.esm.js:1935
(anonymous) @ vue-router.esm.js:2309
(anonymous) @ index.js:195
iterator @ vue-router.esm.js:2287
step @ vue-router.esm.js:1934
runQueue @ vue-router.esm.js:1942
confirmTransition @ vue-router.esm.js:2317
transitionTo @ vue-router.esm.js:2190
init @ vue-router.esm.js:2910
beforeCreate @ vue-router.esm.js:1269
invokeWithErrorHandling @ vue.common.dev.js:1859
callHook @ vue.common.dev.js:4216
Vue._init @ vue.common.dev.js:4994
Vue @ vue.common.dev.js:5074
./resources/js/app.js @ app.js:115
__webpack_require__ @ bootstrap:63
0 @ app.js:150423
__webpack_require__ @ bootstrap:63
(anonymous) @ bootstrap:198
(anonymous) @ bootstrap:198
vue.common.dev.js:1893 Error: ResponsiveGridLayout: `cols` entry for breakpoint null is missing!
    at getColsFromBreakpoint (vue-grid-layout.common.js:3305)
    at VueComponent.mounted (vue-grid-layout.common.js:11612)
    at invokeWithErrorHandling (vue.common.dev.js:1859)
    at callHook (vue.common.dev.js:4216)
    at Object.insert (vue.common.dev.js:3142)
    at invokeInsertHook (vue.common.dev.js:6337)
    at VueComponent.patch [as __patch__] (vue.common.dev.js:6554)
    at VueComponent.Vue._update (vue.common.dev.js:3945)
    at VueComponent.updateComponent (vue.common.dev.js:4063)
    at Watcher.get (vue.common.dev.js:4474)
```

**Comment by gmsa** [comment]
Yes, it's a regression I found out with this new release. Will fix it tomorrow.

**Comment by gmsa** [comment]
fixed in 2.3.11
